evaluators/text/BleuEvaluator/README.md

```diff
@@ -0,0 +1,18 @@
+# BleuEvaluator
+
+Bilingual Evaluation Understudy Score. Evaluates the generated sentence against a desired sentence. 
+A perfect match will score 1.0 and a complete mismatch will score 0.0
+
+BLEU works well with n-grams of at least 4, if less than that, it's neccessary to use a smoothing function and reset the weights.
+If this is not done, it can be the case that we get results smaller than zero which shouldn't happen, as well as returning the error
+
+```
+ZeroDivisionError: Fraction(0, 0)
```

Comment:
Do we have a case where we raise this exception? if so , we should test it

Reply:
It won't get there because we are setting the weights according to the size of the n_gram, so it will be checked in `get_score` and use only the correct weights

Reply:
So better remove the comment?

Reply:
It was more to explain why I was doing the checks, but sure you're right, it makes no sense